lib/notifiers/notifiers.go

```diff
@@ -216,10 +217,12 @@ func Main(notifier Notifier) error {
 		return fmt.Errorf("failed to call SetUp on notifier: %w", err)
 	}
 
+	_, ignoreBadMessages := GetEnv("IGNORE_BAD_MESSAGES")
```

Comment:
What does happen when this env variable doesn't exist?
And how does it know it's a boolean?

Reply:
`GetEnv` returns (string, bool) -- the bool is true iff the string is nonempty. I think we can just document this as "If you want to enable this feature, just set `IGNORE_BAD_MESSAGES=true`". Does that sound ok?

Reply:
So if `IGNORE_BAD_MESSAGES = false`, it won't do what is expected by the user?

Reply:
They just have to unset it (`IGNORE_BAD_MESSAGES=` or remove it from the environment all together).